tests/migrations/test_runner.py

```diff
@@ -197,13 +199,13 @@ def get_sampling_key() -> str:
     runner._update_migration_status(
         MigrationKey(MigrationGroup.TRANSACTIONS, "0001_transactions"), Status.COMPLETED
     )
-    runner.run_migration(
-        MigrationKey(
-            MigrationGroup.TRANSACTIONS,
-            "0002_transactions_onpremise_fix_orderby_and_partitionby",
-        ),
-        force=True,
-    )
+    transaction_migrations = get_group_loader(
+        MigrationGroup.TRANSACTIONS
+    ).get_migrations()
+    for migration in transaction_migrations[1:]:
+        runner.run_migration(
+            MigrationKey(MigrationGroup.TRANSACTIONS, migration), force=True,
+        )
```

Comment:
Why make this change? Since we're only testing the 2nd migration which is the complex one, it shouldn't depend on running the subsequent ones as well

Reply:
Sorry this was a hold out when I changed the test in an earlier iteration, I'll revert.